plugins/lsp/default.nix

```
@@ -206,12 +207,13 @@ in
 
                 options = {
                   inherit (cfg.keymaps) silent;
+                  inherit desc;
```

```suggestion
                  desc = lib.mkDefault desc;
```
Maybe we should do that so that the user can override the description without having to use `lib.mkForce`.

---

Actually, not sure this is needed? If the user has defined a `desc` it'll be part of the `actionProps` that overrides anything set here, right?

---

> If the user has defined a `desc` it'll be part of the `actionProps` that overrides anything set here, right?

Exactly!